teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Person'
+
+class Teacher < Person
+  def initialize(age:, specialization:, name: 'Unknown', parent_permission: true)
+    super(name: name, parent_permission: parent_permission, age: age)
```

Comment:
- [ ] Same here kindly consider removing unnecessary values

```suggestion
    super(name, parent_permission, age)
```